### DISCUSSION TITLE
Is the Converter meant to do this to Unity?

### DISCUSSION THREAD

**Discussion by ZoraPotato44** [esperecyan/VRMConverterForVRChat#40] [Q&A] [answered]
I don't know how to fix I'm a noob at unity xD is there a problem with my computer is unity not intelligent enough?
![image](https://user-images.githubusercontent.com/70824760/173739865-86d03806-d7f2-4543-9280-8fb190822710.png)
![image](https://user-images.githubusercontent.com/70824760/173739910-72d3eda0-2662-4043-82e7-7a9560a2a197.png)

**Comment by esperecyan** [comment, marked as answer]
VRM Converter for VRChat is a package (UPM), not an asset, so it does not exist under the Assets folder.

If you want to check if it has been installed properly, please check the package manager.

**Reply by ZoraPotato44** [reply]
I will check with the package manager in unity thanks for the help 😁
